apps/pie-docs/src/assets/styles/components/_card.scss

```diff
@@ -55,11 +55,11 @@ h2 + .c-card-wrapper {
     justify-content: space-between;
 
     &:hover {
-        background-color: darken(f.$color-container-default, f.$color-hover-01);
-
-        .c-card-image {
-            filter: brightness(50%);
-        }
+        // This is really annoying, but it's the only way to get darkness to work with Custom Props
+        // We should likely make a utility function for this in the future
+        // For ref, I tried using string interpolation with -#{f.$color-hover-01}, but it doesn't work as
+        // CSS then complains about it then not being a number ¯\_(ツ)_/¯
+        background-color: color.adjust(f.$color-container-default, $lightness: calc(f.$color-hover-01 * -1));
```

Comment:
This doesn't seem so bad to me? I don't think the comment is super necessary

Reply:
It's mainly to remind myself to add it as a utility later – once we do that, I'll remove it 😄

Reply:
I'd suggest using `// TODO: ...` for being easy to search. VSCode has extensions that index and highlight to-dos.

<img width="1440" alt="image" src="https://github.com/justeattakeaway/pie/assets/448801/8ab710df-d2b6-44d4-9a37-c3faaf87c1fd">

Reply:
I've added this as a TODO now 👍